engine/table/src/main/java/io/deephaven/engine/table/impl/lang/QueryLanguageParser.java

```diff
@@ -465,6 +471,49 @@ private Class<?> findNestedClass(Class<?> enclosingClass, String nestedClassName
         return m.get(nestedClassName);
     }
 
+    private Method getMethod(final Class<?> scope, final MethodCallExpr methodCallExpr, final Class<?>[] paramTypes,
+            final Class<?>[][] parameterizedTypes) {
+        final String methodName = methodCallExpr.getNameAsString();
+        final QueryScope queryScope = ExecutionContext.getContext().getQueryScope();
+
+        if (scope == null) {
+            final Class<?> methodClass = variables.get(methodName);
+            if (methodClass == PyCallableWrapper.class) {
+                final Object paramValueRaw = queryScope.readParamValue(methodName, null);
+                final PyCallableWrapper pyCallableWrapper = (PyCallableWrapper) paramValueRaw;
+                try {
+                    String javaMethodName = pyCallableWrapper.getAttribute("_j_simple_name", String.class);
+                    methodCallExpr.setName(javaMethodName);
+                    return getMethod(null, javaMethodName, paramTypes, parameterizedTypes);
```

Comment:
it's pretty handy that we don't need to know what dispatch the python/jpy wiring would have done, that was a concern i had reading the python, but not a problem!